src/pages/signup.js

```diff
@@ -32,12 +34,14 @@ const SignUp = () => {
   const emailInputRef = useRef(null)
   const passwordInputRef = useRef(null)
 
-  const [form, setForm] = useState({email: "",
+  const [form, setForm] = useState({
+    email: "",
     password: "",
     message: { text: "", type: "" },
     passwordStrong: false,
     isLoading: true,
-    onVerifyLink: false})
+    onVerifyLink: false,
+  })
```

Comment:
Codacy found an issue: [Unexpected line break before this closing brace.](https://app.codacy.com/gh/treetoplearning/heartwood/pullRequest?prid=6000161)